readme.md

```diff
@@ -58,6 +60,42 @@ The second (optional) argument may include the properties below.
  * `createCwd` - `boolean` Create a directory for `process.cwd()`.  This is `true` by default.
  * `createTmp` - `boolean` Create a directory for `os.tmpdir()`.  This is `true` by default.
 
+### Loading real files & directories
+
+You can load real files and directories into the mock system using `mock.load()`
+
+#### Notes
+
+- All stat information is duplicated (dates, permissions, etc) 
+- By default, all files are lazy-loaded, unless you specify the `{ lazyLoad: false }` option
+
+#### <a id='mappingoptions'>options</a>
+
+| Option    | Type    | Default | Description |
+| --------- | ------- | ------- | ------------
+| lazyLoad  | boolean | true    | File content isn't loaded until explicitly read
```

Comment:
I think this could just be called `lazy` - avoiding a bit of repetition: `load(dir, {lazy: true})`.